### DISCUSSION TITLE
Office_S_Tips/pki/2019/09/19/Podpisywanie_e_dokumentow_pz_gov_pl

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#16] [Komentarze]
# Office_S_Tips/pki/2019/09/19/Podpisywanie_e_dokumentow_pz_gov_pl

Za pomocą profilu zaufanego pz.gov.pl można można m.in. podpisywać cyfrowo swoje dokumenty, pobrać je i przesyłać/załączać do swojej e-korespondencji.

https://andrzejq.github.io/Office_S_Tips/pki/2019/09/19/Podpisywanie_e_dokumentow_pz_gov_pl.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
